pkgs/by-name/sp/speechd/package.nix

```diff
@@ -70,6 +70,10 @@ stdenv.mkDerivation (finalAttrs: {
     })
   ];
 
+  postPatch = lib.optionalString withPico ''
```

Comment:
I generally prefer declarative then imperative grouping as described in https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/attribute-ordering.md

The imperative stuff would ideally disappear eventually.